dutch/bos/donker-bos.md

```diff
@@ -4,3 +4,5 @@ Je verstaat geen woord ervan, maar de stemmen komen wel steeds sneller naar je t
 Je verstopt je in de struiken, en ziet twee mannen met een lijkzak sleuren.
 
 Je gaat snel [terug naar de hei](../welkom.md)
+
+Je gaat [de confrontatie](bos/confrontatie.md) aan, ookal heb je enkel een slappe zwemnoodle om je te verdedigen. 
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.